content/contributing/contribute-code.md

```diff
@@ -1,35 +1,29 @@
 # Contribute Code to the XRP Ledger
 
-The software that powers the XRP Ledger is open-source, so anyone can download, modify, extend, or explore it. Some repositories you may want to consider include the following:
-
-| Repository                      | Language       | Description               |
-|:--------------------------------|:---------------|:--------------------------|
-| [rippled][]                     | C++            | The core XRP Ledger server. |
-| [ripple-lib][]                  | TypeScript     | JavaScript client library. |
-| [ripple-address-codec][]        | TypeScript     | XRP Ledger address encoding/decoding library. |
-| [ripple-binary-codec][]         | TypeScript     | XRP Ledger binary data encoding/decoding library. |
-| [ripple-keypairs][]             | TypeScript     | XRP Ledger elliptic curve cryptography library. |
-| [rippled-historical-database][] | JavaScript     | Data API server and XRPCharts backend. |
-| [standards-drafts][]            | N/A            | Standards and specifications for XRP Ledger software. |
-| [Xpring4j][]                    | Java           | Java SDK for XRP Ledger, Interledger, and PayID. |
-| [XpringKit][]                   | Swift          | Swift SDK for XRP Ledger, Interledger, and PayID. |
-| [Xpring-JS][]                   | TypeScript     | JavaScript SDK for XRP Ledger, Interledger, and PayID.
-| [XRP-API][]                     | TypeScript     | REST API to the XRP Ledger. |
-| [xrpl-dev-portal][]             | HTML, Markdown | This website.             |
-| [XUMM-App][]                    | TypeScript     | Client application and developer platform for XRP Ledger. (Not freely licensed for all purposes.) |
-
-[rippled]: https://github.com/ripple/rippled
-[ripple-lib]: https://github.com/ripple/ripple-lib
-[ripple-address-codec]: https://github.com/ripple/ripple-address-codec
-[ripple-binary-codec]: https://github.com/ripple/ripple-binary-codec
-[ripple-keypairs]: https://github.com/ripple/ripple-keypairs
-[rippled-historical-database]: https://github.com/ripple/rippled-historical-database
-[standards-drafts]: https://github.com/xrp-community/standards-drafts
-[Xpring4j]: https://github.com/xpring-eng/xpring4j
-[XpringKit]: https://github.com/xpring-eng/xpringkit
-[Xpring-JS]: https://github.com/xpring-eng/xpring-js
-[XRP-API]: https://github.com/xpring-eng/xrp-api
-[xrpl-dev-portal]: https://github.com/ripple/xrpl-dev-portal
-[XUMM-App]: https://github.com/XRPL-Labs/XUMM-App
+The software that powers the XRP Ledger is open-source, so anyone can download, modify, extend, or explore it. Look for "[Source]" links in the [documentation](docs.html) to jump directly into the related source code, or browse the source code on GitHub:
```

Comment:
I'd love some phrase that encourages participation. I'm not sure what. Maybe: "...Ledger is open-source. Community participation makes it stronger. Anyone can download..."